jlm/rvsdg/graph.cpp

```diff
@@ -24,6 +24,20 @@ GraphImport::debug_string() const
   return util::strfmt("import[", Name_, "]");
 }
 
+GraphImport &
+GraphImport::Copy(Region & region, StructuralInput *)
+{
+  return Create(*region.graph(), Type(), Name());
+}
+
+GraphImport &
+GraphImport::Create(Graph & graph, std::shared_ptr<const rvsdg::Type> type, std::string name)
+{
+  auto graphImport = new GraphImport(graph, std::move(type), std::move(name));
+  graph.GetRootRegion().append_argument(graphImport);
```

Comment:
Is it necessary to append the argument even when the `GraphImport` constructor passes the root region to the `RegionArgument` constructor? If so, why is it not also done in the `Copy` method?